### DISCUSSION TITLE
DataTable: How to render the expand icon for expanded and closed row?

### DISCUSSION THREAD

**Discussion by topvis** [tusen-ai/naive-ui#3699] [Q&A]
The default expand icon rotate 90 degree for expanded row like shown in the screenshot:
![image](https://user-images.githubusercontent.com/5996531/189269170-b8fbf560-73cf-40ec-a490-227f10b96201.png)

How to render customized expand icons using DataTable's API `render-expand-icon    () => VNodeChild` to show different icons (or different angles) for expanded and closed row?

**Comment by 07akioni** [comment]
Currently it's not possible. We may add a feature to solve the problem later.
